src/lane_follow/package.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<package>
+  <name>lane_follow</name>
+  <version>0.0.0</version>
+  <description>Branch dedicated to detecting lanes using the Hough Transform and staying in lane accordingly</description>

```

Comment:
just for readability, this should be more like:
```
<description>
    Branch dedicated to detecting lanes using the Hough Transform and staying in lane accordingly
</description>
``` 
that way if it gets longer, we can just do some neat line breaks and still have it pretty readable.